axis_utils/include/axis_utils2.inc

```diff
@@ -242,17 +242,17 @@
        endif
     enddo
 
-    if (value < array(1) .or. value > array(ia)) then
- !       if (value < array(1))  frac_index = 1.
- !       if (value > array(ia)) frac_index = float(ia)
+    if (rval < array(1) .or. rval > array(ia)) then
+ !       if (frac_value < array(1))  frac_index = 1.
+ !       if (frac_value > array(ia)) frac_index = float(ia)
```

Comment:
Do you need to change these too?

Reply:
those are comments so I must have not noticed them, good catch. updated in 2e22913c

Reply:
yes sir, removed in 77e6d840